src/substrate-bridge.md

```diff
@@ -0,0 +1,224 @@
+
+Root extrinsics: `initialize`, `addPeer`, `removePeer`.
+
+- **The Relayer**: Developed by SORAMITSU ([source code](https://github.com/sora-xor/sora2-network/tree/master/relayer)). The Relayer is a separate service and not a pallet. It plays a crucial role in the Federated Bridge, passing through messages and holding private keys for signatures, similar to the Beefy bridge implementation.
+
+## Guides
+
+### Transfers
+
+::: warning
+
+Only specific XCM messages for/from the corresponding networks have been tested and utilised in the client applications, such as Polkaswap. When constructing your own messages, exercise caution: there is a possibility that your funds could be permanently lost.
+
+:::
+
+#### SORA mainnet -> Relaychain / Other Prachain
+
+To execute a cross-chain token transfer, a registration process is required beforehand. Token registration involves using the root extrinsics `ParachainBridgeApp.registerSidechainAsset` and `ParachainBridgeApp.registerThischainAsset`. For instance, to register a native token on the Rococo network, the following procedure is followed:
+
+![Image: Example of registering a Rococo native token](./assets/sidechain-token-registration.png)
+
+Similarly, to register a native token from another blockchain as transferable through a parachain, the process is as follows:
+
+![Image: Example of registering a native blockchain for transfer via parachain](./assets/thischain-token-registration.png)
+
+These extrinsics send messages to the SORA 2 Parachain, enabling the registration of asset mappings. Without this registration, tokens cannot be transferred through the parachain.
+
+Once the token is successfully registered, transfers can be initiated. To transfer a token to a relaychain, the extrinsic `ParachainBridgeApp.burn` is used. For instance, to transfer a ROC token to the Rococo relay chain, the following procedure is followed:
+
+![Image: Example of transferring ROC to the Rococo relay chain](./assets/token-burn.png)
+
+To specify a parachain as the transfer destination, the following request needs to be made:
+
+![Image: Example of indicating a parachain as the transfer target](./assets/transfer-to-parachain.png)
+
+By following these steps, cross-chain transfers can be successfully executed, allowing for seamless movement of tokens between different chains within the network.
+
+### How to Track the Status of the Transfer
+
+#### Other Parachain -> SORA mainnet
+
+Example extrinsic: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws.karura-node-1.c2.dev.sora2.soramitsu.co.jp#/extrinsics/decode/0x3600008000407a10f35a00000000000000000000010102006d1f0100e6cacc509b47920b3fe9329224df74640cd6861d40132633aae46c168ab73e4d00
```

Comment:
I think we should not share private dev environment enpoints in docs

Reply:
https://github.com/sora-xor/sora-docs/pull/44/commits/d668f091ec53b2338324515c5a5be1d2a19302fe